File: frontend/src/app/classes/review.ts
import { EventClass } from "./eventClass";

export class Review {

    
    
    comment: string;
    rating:number;
    forWhatEvent:EventClass;
    id?: number;
   
    

    constructor(      
        
        Comment: string,
        Rating:number,
        ForWhatEvent:EventClass,
        Id?: number
    ) 
    {        
        
        this.comment=Comment;
        this.rating=Rating;
        this.forWhatEvent=ForWhatEvent;
        this.id=Id;
    }
}

File: backend/Models/Review.cs
using System.ComponentModel.DataAnnotations;

namespace Models
{
    public class Review
    {
        [Key]
        public int ID { get; set; }
        
        [Required]
        public string Comment { get; set; }

        [Required]
        public int Rating { get; set; }

        [Required]
        public Event ForWhatEvent {get; set;}


    }
}